components/places/src/places.udl

```diff
@@ -93,6 +143,38 @@ dictionary HistoryHighlight {
     string? preview_image_url;
 };
 
+dictionary HistoryVisitInfo {
+    string url;
+    string? title;
+    Timestamp timestamp;
+    i32 visit_type;
```

Comment:
sorry I didn't pick this up before - this should be an enum - but I think lougenia's patch is adding to the the .udl - so maybe add an `XXX` comment to change it?